app/_components/commodities/components/GridBodyItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { formatThousands } from '@/app/_hooks/textFormatting';
+import { GridItem, SimpleGrid } from '@chakra-ui/react';
+import { ICommodityFormResponse } from '@/app/_types/commodity';
+import selectColor from '@/app/_hooks/fontColorSelector';
+import useColorMode from '@/app/_hooks/useColorMode';
+import { calculateTimeDifference, renderStationTypeIcon } from '../helpers';
+
+interface IGridBodyItemProps {
+  commodity: ICommodityFormResponse;
+  isBuying: boolean;
+}
+
+const GridBodyItem: React.FC<IGridBodyItemProps> = ({
+  commodity,
+  isBuying,
+}) => {
+  const { isDark } = useColorMode();
+
+  return (
+    <SimpleGrid
+      columns={[5, 7]}
+      width="100%"
+      gap={8}
+      fontSize="sm"
+      _odd={{
+        background: `${selectColor(isDark, '')}`,
```

Comment:
Also the alt row colouring in light mode I thik is the same as the background colour so is only visble over the top of the background image

Reply:
These rows definitely need some better contrast. Putting a pin in it temporarily until the color scheme config files are updated. I'll follow up with you w/ some screen grabs once we've updated the project's colors and get your feedback on the tables then. That way we can improve on a version closer to final. If we accomplish a color update soon then this merge can wait until that's completed.

Reply:
fixed

![Screenshot 2023-08-29 at 16-30-00 EDPN - Commodities](https://github.com/ed-pilots-network/frontend/assets/67390534/73677f88-ecd5-4ba1-92f4-6a86afb91e68)